src/lib/components/admin/Settings/Models/Manage/ManageOllama.svelte

```
@@ -0,0 +1,1062 @@
+
+			MODEL_DOWNLOAD_POOL.set({
+				...$MODEL_DOWNLOAD_POOL,
+				[sanitizedModelTag]: {
+					...$MODEL_DOWNLOAD_POOL[sanitizedModelTag],
+					abortController: controller,
+					reader,
+					done: false
+				}
+			});
+
+			while (true) {
```

ESLint

[error] Unexpected constant condition.

(no-constant-condition)